Calibration/EcalCalibAlgos/src/EcalPhiSymRecHit.cc

```diff
@@ -0,0 +1,60 @@
+#include "Calibration/EcalCalibAlgos/interface/EcalPhiSymRecHit.h"
+
+//**********constructors******************************************************************
+EcalPhiSymRecHit::EcalPhiSymRecHit()
+    : id_(0), chStatus_(0), nHits_(0), etSum_(1, 0.), et2Sum_(0), lcSum_(0), lc2Sum_(0) {}
+
+EcalPhiSymRecHit::EcalPhiSymRecHit(uint32_t id, unsigned int nMisCalibV, unsigned int status)
+    : id_(id), chStatus_(status), nHits_(0), etSum_(nMisCalibV, 0.), et2Sum_(0), lcSum_(0), lc2Sum_(0) {}
+
+EcalPhiSymRecHit::EcalPhiSymRecHit(uint32_t id, std::vector<float>& etValues, unsigned int status)
+    : id_(id),
+      chStatus_(status),
+      nHits_(0),
+      etSum_(etValues.begin(), etValues.end()),
+      et2Sum_(0),
+      lcSum_(0),
+      lc2Sum_(0) {}
+
+//**********destructor********************************************************************
+EcalPhiSymRecHit::~EcalPhiSymRecHit() {}
+
+//**********utils*************************************************************************
+
+void EcalPhiSymRecHit::AddHit(float* etValues, float laserCorr) {
```

Comment:
```suggestion
void EcalPhiSymRecHit::AddHit(const float& etValues, const float laserCorr) {
```

Reply:
This won't work I think, since etValues is an array.

Reply:
Could still be `const`. At least `laserCorr`.

Reply:
I removed the method anyway